hawk/app/views/sessions/new.html.haml

```diff
@@ -3,4 +3,4 @@
   = main_form.password_field :password, placeholder: _("Password"), label: { class: 'sr-only' }
 
   = main_form.submit _("Login"), class: "btn btn-primary btn-block"
-  = link_to _("Cyphre Central"), "/", class: "btn btn-default btn-block"
+  = link_to _("Cyphre Central"), "/", class: "btn btn-ccui btn-block"
```

Comment:
I think what you mean to do is remove the class altogether so it's not a button at all and then you wont need the css